apps/teams-test-app/src/components/SearchAPIs.tsx

```diff
@@ -0,0 +1,35 @@
+import { search } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+//import { search } from '../../../../packages/teams-js/src/public/search';
+import { ApiWithoutInput } from './utils';
+
+const RegisterHandlers = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'search_registerHandlers',
+    title: 'Search Register Handlers',
+    onClick: async setResult => {
+      const onChange = (onChangeHandler: search.SearchQuery): void => {
+        console.log(onChangeHandler.searchTerm);
+      };
+      const onClosed = (onClosedHandler: search.SearchQuery): void => {
+        console.log(onClosedHandler);
```

Comment:
you can't validate against these in our e2e scenario tests. This should print something on the screen itself within the tests app so you can assert on it. you could use `setResult` for that, maybe?